common/src/main/java/com/wynntils/commands/CompassCommand.java

```diff
@@ -271,7 +271,7 @@ public int territory(CommandContext<CommandSourceStack> context) {
         Models.Marker.USER_WAYPOINTS_PROVIDER.addLocation(location.asLocation());
 
         MutableComponent response = Component.literal(
-                        "Compass set to middle of " + territoryProfile.getFriendlyName() + " at ")
```

Comment:
Why?! Revert this. Getters should be getters.

Have you really used the Artemis or Artemis mini analysis profiles?

Reply:
this changed because TerritoryProfile is now a record. revert?